packages/edit-site/src/components/page-library/grid-item.js

```diff
@@ -52,7 +52,6 @@ export default function GridItem( { categoryId, composite, icon, item } ) {
 		postId: item.type === USER_PATTERNS ? item.id : item.name,
 		categoryId,
 		categoryType: item.type,
-		canvas: 'view',
```

Comment:
As long as when we go through to a pattern the sidebar is shown then we're all good.